packages/testing/src/api/index.ts

```diff
@@ -15,10 +15,18 @@
  */
 
 import { firebase } from '@firebase/app';
-import request from 'request-promise';
+import * as request from 'request';
 import { FirebaseApp, FirebaseOptions } from '@firebase/app-types';
 import { base64 } from '@firebase/util';
 import { setLogLevel, LogLevel } from '@firebase/logger';
+import * as grpc from 'grpc';
+import * as fs from 'fs';
+
+const PROTOS = grpc.load({
+  root: __dirname + '/../protos',
+  file: 'google/firestore/emulator/v1/firestore_emulator.proto'
+});
+const EMULATOR = PROTOS['google']['firestore']['emulator']['v1'];
```

Comment:
are each of these all guaranteed to exist?

Reply:
Kinda. The `protos/` directory is structured such that these do exist. If they don't, it is a build error on our part.